packages/ripple-binary-codec/test/hash.test.js

```diff
@@ -50,10 +50,8 @@ describe('Hash256', function () {
 })
 
 describe('Currency', function () {
-  test('Will throw an error for dodgy XRP ', function () {
-    expect(() =>
-      Currency.from('0000000000000000000000005852500000000000'),
-    ).toThrow()
+  test('Decoding allows dodgy XRP without throwing', function () {
+    Currency.from('0000000000000000000000005852500000000000')
```

Comment:
This test isn't checking anything

Reply:
If the test throws it fails right? (I just want to make sure that the decoding happens without any exceptions being raised)

Reply:
You should still check the result in a test - otherwise it's not a complete test